Expo1900/Expo1900/Model/KoreanEntries.swift

```diff
@@ -0,0 +1,15 @@
+//  KoreanEntries.swift
+//  Created by Mangdi, zhilly on 2022/10/17
+
+struct KoreanEntries: Codable {
+    let name: String
+    let imageName: String
+    let shortDesc: String
+    let desc: String
```

Comment:
`Desc`, `desc`는 축약형으로 보여집니다.
[Swift API Design Guideline](https://www.swift.org/documentation/api-design-guidelines/#naming)에서는 
**Include all the words needed to avoid ambiguity** , **Omit needless words.** 라는 구절이 있어요.
둘 중 어느부분에 해당시키는게 좋을지 한번 고민해 봅시다.

Reply:
태태의 말에 동의합니다!
shortDesc -> shortDescription
desc -> description
으로 변경했습니다